tests/common/ixia/ixia_fixtures.py

```diff
@@ -11,6 +11,7 @@
 from tests.common.ixia.common_helpers import get_vlan_subnet, get_addrs_in_subnet,\
     get_peer_ixia_chassis
 from tests.common.ixia.ixia_helpers import IxiaFanoutManager, get_tgen_location
+import snappi
 
 try:
     from abstract_open_traffic_generator.port import Port
```

Comment:
We need to get rid of `abstract_open_traffic_generator` everywhere.

Reply:
And statements using abstract classes

Reply:
Added TODO as discussed